packages/grid/data-grid/src/tests/toolbar.DataGrid.test.tsx

```diff
@@ -55,7 +55,14 @@ describe('<DataGrid /> - Toolbar', () => {
       const rowHeight = 30;
       const { getByText } = render(
         <div style={{ width: 300, height: 300 }}>
-          <DataGrid {...baselineProps} showToolbar rowHeight={rowHeight} />
+          <DataGrid
+            {...baselineProps}
+            showToolbar
```

Comment:
I thought we wanted to remove showToolbar

Reply:
See https://github.com/mui-org/material-ui-x/pull/971#discussion_r568587680